### PR TITLE
mset() and mget() accept variadic parameters

### PR DESCRIPTION
**Pull request by OddKristensen**

### CODE REVIEW

File: core/src/main/scala/io/chrisdavenport/rediculous/RedisCommands.scala [hunk 1]
```diff
@@ -696,11 +696,15 @@ object RedisCommands {
   def expire[F[_]: RedisCtx](key: String, seconds: Long): F[Boolean] = 
     RedisCtx[F].keyed(key, NEL.of("EXPIRE", key.encode, seconds.encode))
 
-  def mget[F[_]: RedisCtx](key: String): F[List[Option[String]]] = {
+  private[this] def mget[F[_]: RedisCtx](key: String): F[List[Option[String]]] = {
     val cmd = NEL("MGET", key.encode :: Nil)
     RedisCtx[F].keyed(key, cmd)
   }
 
+  def mget[F[_]: RedisCtx](key: String, keys: String*): F[List[Option[String]]] = {
```
Comment (Kazark) [inline core/src/main/scala/io/chrisdavenport/rediculous/RedisCommands.scala:704]: @ChristopherDavenport I told him that I thought this `(head, tail*)` pattern was necessary to disambiguate the overloads. Let me know if I'm wrong there.

**Review by Kazark — APPROVED** [review]
Looks good to me. Chris'll have to tell us about whether `private[this]` makes sense.

**Comment by ChristopherDavenport** [comment]
`private[this]` is actually more private than `private`  if my understanding serves this needs to be package private.

**Comment by Kazark** [comment]
That makes sense.

**Comment by OddKristensen** [comment]
> `private[this]` is actually more private than `private` if my understanding serves this needs to be package private.

💯 , done.

**Review by Kazark — APPROVED** [review]

**Pull request merged**